test/test_linalg.py

```diff
@@ -6165,6 +6165,41 @@ def genf_float(x, y):
 
             _test_mm(n, m, p, dtype, genf)
 
+    @onlyOnCPUAndCUDA
+    def test_mm_bmm_non_memory_dense(self, device):
+        def _slice(tensor, fn):
+            return fn(tensor)[:, ::2]
```

Comment:
```suggestion
        def _slice(tensor, fn):
            return fn(tensor)[..., ::2]
```
this can be used for batched tensor also